src/OpenTK.Windowing.Desktop/NativeWindow.cs

```diff
@@ -24,6 +24,8 @@ namespace OpenTK.Windowing.Desktop
     /// </summary>
     public class NativeWindow : INativeWindow
     {
+        private const int JoystickStatesCount = 16;
```

Comment:
Just personal preference, but I would put this constant closer to the array declarations to indicate association.